openshift_cli_installer/libs/clusters/ocp_cluster.py

```diff
@@ -139,6 +144,19 @@ def _add_s3_bucket_data(self):
             f"{self.cluster_info['name']}-{self.cluster_info['shortuuid']}.zip"
         )
 
+    def check_and_assign_aws_cluster_region(self):
+        if not self.cluster_info.get("region"):
+            if self.cluster_info["platform"] in [AWS_STR, AWS_OSD_STR]:
+                region, vpcs = 0, 0
```

Comment:
duplicate code (https://github.com/RedHatQE/managed-services-integration-tests/blob/main/tests/hypershift/test_hypershift.py#L39); please extract to cloud-tools and use in both places  (e.g `get_least_crowded_vpc_region`)